src/main/java/com/google/devtools/build/lib/rules/proto/ProtoLangToolchain.java

```diff
@@ -37,6 +37,7 @@
   @Override
   public ConfiguredTarget create(RuleContext ruleContext)
       throws InterruptedException, RuleErrorException, ActionConflictException {
+    ProtoCommon.checkRuleHasValidMigrationTag(ruleContext);
```

Comment:
Please add a test case to `BazelProtoLibraryTest`.

Reply:
Done (+`ProtoLangToolchainTest`)